app/controllers/api/v1/requests_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class RequestsController < ApplicationController
+      before_action :authenticate_api_v1_user!
+
+      def create
+        @request = current_api_v1_user.requests.create!(request_params)
+
+        render status: :created
+      end
+
+      private
+
+      def request_params
+        params.require(:request).permit(:text)
```

Comment:
WDYT of naming it `message`? It feels a bit more specific of what the variable has on it